pachyderm/values.schema.json

```diff
@@ -18,12 +21,42 @@
                             "type": "string"
                         }
                     }
+                },
+                "resources": {
```

Comment:
Will this work if we pass an empty object for resources? (ie. disabling resources)

Reply:
Yes (with the guards in commit c6b0978), but per https://helm.sh/docs/chart_template_guide/values_files/#deleting-a-default-key one needs to set it to `null`, not `{}`.